.github/workflows/systest.yml

```diff
@@ -34,9 +34,9 @@ env:
   CI_REGION_NAME: ${{ secrets.CI_REGION_NAME }}
   USE_GKE_GCLOUD_AUTH_PLUGIN: True
 
-concurrency:
-  group: ${{ github.base_ref == 'staging' && 'smci-staging' || format('smci-{0}-{1}', github.workflow, github.ref) }}
-  cancel-in-progress: ${{ github.base_ref == 'staging' && false || true }}
+#concurrency: # XXX:  Check if we need this
+#  group: ${{ github.base_ref == 'staging' && 'smci-staging' || format('smci-{0}-{1}', github.workflow, github.ref) }}
+#  cancel-in-progress: ${{ github.base_ref == 'staging' && false || true }}
```

Comment:
Yes, there should only be one merge in progress and other should block until the merge completed